config/locales/devise.en.yml

```diff
@@ -0,0 +1,65 @@
+# Additional translations at https://github.com/heartcombo/devise/wiki/I18n
```

Comment:
Remember to keep adding to the translations.

Reply:
Ok, I will have them in mind.